### ISSUE TITLE
[Windows][4.4.0]: clicking on notification doesn't open channel if app window isn't active

### ISSUE THREAD

**Issue by PetbkA** [mattermost/desktop#1217]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] This is not a troubleshooting question. [Troubleshooting questions go here: http://www.mattermost.org/troubleshoot/](http://www.mattermost.org/troubleshoot/).
- [X] This doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**
Starting from _4.4.0_, clicking on desktop notification only closes notification without opening app window, if latter _isn't active_. The bug persists event if app window is open, but not active.

**Environment**
- Operating System: Windows 10 x64 LTSC with last OS Updates;
- Mattermost Desktop App version: 4.4.0;
- Mattermost Server version: 5.13.2.

**Steps to reproduce**
1. Switch to non-MM window or minimize or close MM so it was minimized to tray;
2. Trigger desktop notification, e.g. by asking someone to send you a message;
3. Left click on notification.

**Expected behavior**
MM window activates or opens and switches to channel from which the message was sent.

**Observed behavior**
Notification closes, MM windows doesn't open, doesn't activate itself and doesn't switch channel.

Bug appeared in client ver. _4.4.0_, everything was OK in _4.2.1_ or _4.3.2_. Server version was always the same, nothing was changed to OS Notifications settings and Background apps settings. Works OK in browser (tested with _Chrome 80.0.3987.122 (Official Build) (64-bit)_)

**Comment by amyblais** [comment]
@PetbkA Are you seeing this issue also on a more recent server version? Our most recent version is v5.20.1.

**Comment by PetbkA** [comment]
> @PetbkA Are you seeing this issue also on a more recent server version? Our most recent version is v5.20.1.

Right now I can't test it - updating of our server is planned on the end of March

**Comment by PetbkA** [comment]
I've been a little tired of this issue so I've decide то return to _4.3.2_, and then on _4.2.1_... and the issue is still there!

Moreover, one of my colleagues installed _4.3.2_ from scratch and has the issue too. His OS and MM Server are the same as mine. So I guess the issue may be not on the OS side

**Comment by flexo3001** [comment]
An customer is seeing this on Windows 10 as well.

**Comment by PetbkA** [comment]
> An customer is seeing this on Windows 10 as well.

May be you can share client's versions of Windows, MM Desktop App and MM Server?

**Comment by PetbkA** [comment]
We've just recently updated MM server to _5.21_. Colleagues who had problem with notifications and client earlier than _4.3.2_ received notifications from server to update to _4.3.2_, and the problem was gone after the update.

For _4.4.0_ there was no client update offer. I've restarted the client, but the problem was still there. 

Then I did the following:
1. Uninstalled client;
2. Removed:
   1. All HKCU keys with "Mattermost";
   2. "Mattermost" folder from %appdata%;
3. Installed client again.

After that, the problem was gone. I guess that the key step was 2.2, but may be simple reinstall would be enough too
